src/ml/neural_net/mps_layer_conv_padding.h

```diff
@@ -0,0 +1,36 @@
+#ifndef mps_layer_conv_padding_h
+#define mps_layer_conv_padding_h
+
+#import <Accelerate/Accelerate.h>
+#import <Foundation/Foundation.h>
+#import <Metal/Metal.h>
+#import <MetalPerformanceShaders/MetalPerformanceShaders.h>
+
+API_AVAILABLE(macos(10.14))
+@interface ConvolutionPadding : NSObject <MPSNNPadding>
+{
```

Comment:
Let's try to keep the header files focused on the interface as much as possible, and leave implementation details to the .m files. This makes headers easier to read, and these are the files that clients must read most often.

In Objective C, we can omit the instance variables entirely. I would suggest putting this code in the .m file:
```
@interface ConvolutionPadding ()

@property (nonatomic) NSUInteger paddingWidth;
@property (nonatomic) NSUInteger paddingHeight;
@property (nonatomic) NSUInteger strideWidth;
@property (nonatomic) NSUInteger strideHeight;

@end
```

This will auto-synthesize instance variables with the standard naming convention, e.g. `_paddingWidth`.

Reply:
Changed!